src/frontend/main/MainWrapper.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Router, RouteComponentProps } from '@reach/router';
+import * as d3 from 'd3';
+import MapView from './map/MapView';
+import TableView from './TableView';
+
+import SubNav from './SubNav';
+
+interface MainWrapperProps extends RouteComponentProps {
+  responseData: any;
+}
+
+interface mapProperties {
+  city: string;
+  responses: number;
+  fever_no: number;
+  fever_yes: number;
+  fever_slight: number;
+  fever_high: number;
+  cough_no: number;
+  cough_yes: number;
+  cough_mild: number;
+  cough_intense: number;
+  general_wellbeing_fine: number;
+  general_wellbeing_impaired: number;
+  general_wellbeing_bad: number;
+  breathing_difficulties_no: number;
+  breathing_difficulties_yes: number;
+  muscle_pain_no: number;
+  muscle_pain_yes: number;
+  headache_no: number;
+  headache_yes: number;
+  sore_throat_no: number;
+  sore_throat_yes: number;
+  rhinitis_no: number;
+  rhinitis_yes: number;
+  stomach_issues_no: number;
+  stomach_issues_yes: number;
+  sensory_issues_no: number;
+  sensory_issues_yes: number;
+  longterm_medication_no: number;
+  longterm_medication_yes: number;
+  smoking_no: number;
+  smoking_yes: number;
+  corona_suspicion_no: number;
+  corona_suspicion_yes: number;
+  population: number;
+  x: number;
+  y: number;
+}
+
+const cartogramData: mapProperties[] = require('./assets/data/cartogram-coordinates.json');
+
+const MessageContainer = styled.div`
+  text-align: center;
+  margin: 24px 0;
+`;
+
+const Container = styled.div`
+  max-width: 600px;
+  margin: 0 auto;
+`;
+
+const TotalResponses = styled.div`
+  background: ${props => props.theme.white};
+  position: fixed;
+  bottom: 0px;
+  padding: 10px 0 10px 16px;
+  font-size: 14px;
+  font-style: italic;
+  width: 100vw;
+  text-align: left;
+
+  p {
+    margin: 0;
+  }
+
+  @media (min-width: 624px) {
+    padding-left: 0;
+  }
+`;
+
+const MainWrapper = (props: MainWrapperProps) => {
+  const currentPath = props.location!.pathname;
+  const isEmbed = currentPath.includes('/map-embed');
```

Comment:
This works great for what we need at the moment, but would it make the logic clearer to move isEmbed to the route definition?  So `isEmbed` is kind of "hard-coded" to the route instead of doing string matching here? :thinking: 

```
            <MainWrapper path="/*" responseData={data} isEmbed={false} />
            <MainWrapper path="/map-embed/*" responseData={data} isEmbed={true} />
```